droidconKE/chai
# added work from 23

---
### ! Edit this before submitting your PR !(Remove this after editing)

---
name: Added work from 23
about: Create a pull request
label: 'triage me'
Fixes [Referenced Issue #](https://github.com/droidconKE/chai/issues)_.


---
## How can this PR be tested?
> Check it out and run

---

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [X ] Ensure your branch is up to date with main branch.
- [ X] Ensure your branch runs without breaking.
- [X ] Ensure you solved any merge conflicts.
- [  ] Smaller PRs for the ease of reviewing
- [X ] Make sure to open a GitHub issue as a bug/feature request before writing your code! This means that we can discuss the change, evaluate designs, and agree on the general idea
- [X ] Ensure the tests and linter pass (`./gradlew --init-script gradle/init.gradle.kts spotlessApply` to automatically apply formatting)
- [X ] Appropriate docs were updated (You are adding code so...)


Is this your first Pull Request?

Fixes #<issue_number_goes_here> 🦕

# Screenshot
> For PRs that change the UI, attach a before/after screenshot.
> **[REMOVE THIS SECTION IF IT DOES NOT APPLY TO THIS PR]**

![]()
